zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -2169,7 +2169,12 @@ angular.module('zeppelinWebApp').controller('ParagraphCtrl', function($scope, $r
       var row = $scope.paragraph.result.msgTable[r];
       var dsvRow = '';
       for (var index in row) {
-        dsvRow += row[index].value + delimiter;
+        var stringValue =  (row[index].value).toString();
+        if (stringValue.contains(delimiter)) {
+          dsvRow += stringValue.replace(stringValue, '"' + stringValue + '"') + delimiter;
```

Comment:
@meenakshisekar actually agree with felix, same result, so better use the easy one